jest.config.js

```diff
@@ -11,8 +11,8 @@ module.exports = {
     global: {
       branches: 44,
       functions: 42,
-      lines: 48,
-      statements: 48,
+      lines: 53,
+      statements: 53,
```

Comment:
These changes are required because 

```
Basically when the unit test coverage is 5% above the current threshold it fails the build and ask for you to increase the threshold
``` - @PeterYinusa